libphx/src/lua.rs

```diff
@@ -338,12 +338,17 @@ unsafe extern "C" fn Lua_ToString(
     this: *mut Lua,
     name: *const libc::c_char,
 ) -> *const libc::c_char {
+    static_string!(lua_to_string(this, &name.convert()))
+}
+
+unsafe fn lua_to_string(this: *mut Lua, name: &str) -> String {
```

Comment:
This can also be a method on Lua.

Also, `*mut Lua` could be changed to `&mut Lua`.

Reply:
This requires changes in too many places. Let's do it in the next PR

Reply:
Sounds good